pxf/pxf-hive/src/main/java/org/apache/hawq/pxf/plugins/hive/HiveORCAccessor.java

```diff
@@ -213,4 +237,40 @@ private boolean buildArgument(SearchArgument.Builder builder, Object filterObj)
         return true;
     }
 
+    /**
+     * Fetches file-level statistics from an ORC file.
+     */
+    @Override
+    public void retrieveStats() throws Exception {
+        FragmentMetadata fragmentMetadata = Utilities.parseFragmentMetadata(inputData);
+        /*
+         * We are using file-level stats therefore if file has multiple splits,
+         * it's enough to return count for a first split in file.
+         * In case file has multiple splits - we don't want to duplicate counts.
+         */
+        if (inputData.getFragmentIndex() == 0)
+            this.count = this.orcReader.getNumberOfRows();
+    }
+
+    /**
+     * Emits tuple without reading from disk, currently supports COUNT
+     */
+    @Override
+    public OneRow emitAggObject() {
+        OneRow row = null;
+        switch (inputData.getAggType()) {
+            case COUNT:
+                if (objectsEmitted < count) {
+                    objectsEmitted++;
+                    row = new OneRow(key, data);
```

Comment:
as this is likely a dummy object, can you reuse a single object and return it multiple times rather than creating multiple dummy objects ?